templates/form_localbooks.html

```diff
@@ -1,18 +1,18 @@
-<p>Search individual articles</p>
-<form action="https://search.ebscohost.com/login.aspx" id="edssearch" method="get" data-target="eds">
+<form action="https://widgets.ebscohost.com/prod/search/" id="localbooksearch" method="get" data-target="eds">
 	<div class="hidden">
 		<input name="direct" value="true" type="hidden">
-		<input name="scope" value="site" type="hidden">
-		<input name="site" value="eds-live" type="hidden">
 		<input name="authtype" value="ip,guest" type="hidden">
-		<input name="custid" value="s8978330" type="hidden">
-		<input name="profile" value="eds" type="hidden">
+		<input name="type" value="0" type="hidden">
 		<input name="groupid" value="main" type="hidden">
+		<input name="site" value="eds-live" type="hidden">
+		<input name="profile" value="eds" type="hidden">
+		<input name="custid" value="s8978330" type="hidden">
 		<input name="bquery" value="" type="hidden">
+		<input name="facet" value="Books,eBooks,Audiobooks,Dissertations,MusicScores,Audio,Videos" type="hidden">
 	</div>
 	<div class="row">
-		<input type="text" name="uquery" placeholder="Search journal articles...">
-		<input type="submit">
+		<input type="text" name="uquery" placeholder="Search books, films, music scores...">
+		<input type="submit" value="Search">
```

Comment:
Same here: I would rather have the submit semantically last and work on visual arrangement with css. I think you can drop the "row" div and move submit after the limit selector.